app/helpers/auth.bcrypt.js

```diff
@@ -2,15 +2,15 @@ const bcrypt = require("bcrypt");
 const model = require("../../database/models");
 const jwt = require("jsonwebtoken");
 const keys = require("../../utils/keys");
-
+const service = require("../../database/acid/services");
 
 const hashPass = (req, res) => {
     const { body: user } = req;
     const { password } = user;
     bcrypt.genSalt(10, (_, salt) => {
         bcrypt.hash(password, salt, (_, hash) => {
          user["password"] = hash;
-         model.User.create(user)
+         service.createUser(user)
          .then(() => {
              res.status(201).json({
                message: "User created succefully",
```

Comment:
This is not the place to insert the user in the DB. You do this from the controller.